chain-tx-enclave-next/enclave-ra/ra-sp-server/src/main.rs

```diff
@@ -11,7 +11,16 @@ fn main() {
     let config = SpRaConfig::from_args();
     env_logger::init();
 
-    let address = config.address.clone();
-    let server = SpRaServer::new(config).unwrap();
-    server.run(address).unwrap();
+    let mut attempt = 0;
+    while attempt < 3 {
+        let config = SpRaConfig::from_args();
```

Comment:
Can we `clone()` the config instead of parsing from command line arguments repeatedly?

Reply:
will update if it works -- not entirely sure why https://github.com/crypto-com/chain/pull/1933 failed @yihuang